app/controllers/orders_controller.rb

```diff
@@ -22,28 +22,32 @@ def new
   end
 
   def create
-    cart = Cart.new(session[:cart])
+    pending_loans = PendingLoan.new(session[:pending_loan])
     total_cost = 0
-    cart.projects.each do |project, quantity|
-      total_cost += project.price * quantity.to_i
+    pending_loans.projects.each do |project, _loan_amount|
+      total_cost += project.price * loan_amount
     end
     @order = Order.create(
-      user_id: session[:user_id],
+      user_id: session[:user_id].to_i,
       total_cost: total_cost,
-      status: 'ordered'
+      status: "completed"
       )
-    cart.projects.each do |project, quantity|
+    pending_loans.projects.each do |project, loan_amount|
```

Comment:
Please ignore this hound comment.  The diff that the hound is commenting on absolutely has been changed but for some reason the hound isn't acknowledging as much.  Feel free to pull the branch and look in the orders_controller for yourself.